src/azure-cli/azure/cli/command_modules/appservice/tests/latest/sample_custom_container/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
+    <head>
+        <title>Public ACR v1</title>
+        <link rel=\'shortcut icon\' href=\'https://appservice.azureedge.net/images/app-service/v4/favicon.ico\' type=\'image/x-icon\'/>
+    </head>
+    <body bgcolor=lightblue>
+        <h1>Hello Linux from public Azure Container Registry v1</h1>
+    </body>
+</html>
```

Comment:
Why do you need to add this file?

Reply:
The file is used in the unit test to build a custom container with "az acr build"